tests/bgp/test_bgp_speaker_2b4b.py

```diff
@@ -0,0 +1,417 @@
+"""
+This test is for verifying bgp session establishing, route announcing and traffic flow 
+from a 2 byte ASN DUT to 4 byte ASN bgp speakers.
+The script is similar to test_bgp_speaker.py, except that bgp speaker ASN modification.
```

Comment:
please reuse the test_bgp_speaker.py test, do not create a new test. we need to reuse our code. having duplicated code increase the cost of maintainance